accounts/abi/method.go

```diff
@@ -88,9 +88,21 @@ func (method Method) tupleUnpack(v interface{}, output []byte) error {
 	}
 
 	var (
-		value = valueOf.Elem()
-		typ   = value.Type()
+		value    = valueOf.Elem()
+		typ      = value.Type()
+		isStruct = true
```

Comment:
good point. The only additional type which we could support here is map, and this can be easily handled with enum type, which we will define once (performance, checks...)